test/info.py

```diff
@@ -56,6 +56,7 @@
     (   0x5050,     'lpc11u35_arm_watch_stm32f411_if',  None,               None                            ),
     (   0x5051,     'lpc11u35_arm_watch_efm32_if',      None,               None                            ),
     (   0x5052,     'lpc11u35_arm_watch_nrf51_if',      None,               None                            ),
+	(   0x7010,     'lpc11u35_blueninja_if',            None,               'BlueNinja'                     ),
```

Comment:
The target 'BlueNinja' can't be built with the compile API since it is not in mbed-os yet. You'll need to replace 'BlueNinja' with None until support is added. Just leave a comment to the side indicating which target should be used when ready.